e2e/cypress/integration/commands/invalid_command_spec.js

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Please move spec file to `e2e/cypress/integration/integrations` folder, being part of "/Integrations" functional area.